slices/Hero/index.vue

```diff
@@ -40,14 +40,14 @@ defineProps(getSliceComponentProps<Content.HeroSlice>(
         class="flex flex-wrap gap-4"
```

Comment:
The line above this needs to be updated with `v-if="slice.primary.buttons.length"`

Reply:
Oh, nice catch, thanks!